### DISCUSSION TITLE
lazy.nvim Lua module not found for config of nvim. Please use a `config()` function instead

### DISCUSSION THREAD

**Discussion by jottr** [LazyVim/LazyVim#2410] [Q&A] [answered]
I recently started seeing the following error message in nvim:

```lazy.nvim Lua module not found for config of nvim. Please use a `config()` function instead.```  

Sadly, this error message doesn't provide a trace and I'm unsure  where to actually look for the culprit. 
I have used the LazyVim [starter template](https://github.com/LazyVim/starter) as a starting point for my config. 
I'd appreciate any pointers in the general vicinity of what might be the cause or how I could debug this further.

**Comment by dpetka2001** [comment, marked as answer]
You must have a plugin spec in your configuration which has something like for example `catppuccin/nvim` and `lazy.nvim` identifies it as `nvim`. There's something wrong in the configuration of that plugin spec. Need more information for further help after you identify the culprit in your configuration.

**Comment by alewis001** [comment]
I got this message when I tried to configure tweaks to the catppuccin style (I wanted the line numbers to be brighter). I think the problem was that I'd referenced "catppuccin/nvim" in a plugin lua file I'd created, instead of using the name LazyVim defines which is just "catppuccin".

Here's my working lua file in case it helps:
```
return {
  "catppuccin",
  opts = function(_, opts)
    opts.custom_highlights = function(colors)
      return {
        LineNr = { fg = colors.overlay0 },
        CursorLineNr = { fg = colors.crust, bg = colors.yellow },
      }
    end
  end,
}

```

**Reply by jottr** [reply]
The culprit was indeed the "catpuccin" config.
Amazing that you guys figured it out with this little information to go with!
I'm still wondering why nvim doesn't provide a more detailed trace for errors like this.

**Comment by mazunki** [comment]
I fixed this by passing `name = "catppuccin"` to lazy.
